format_spec/array_schema.md

```diff
@@ -14,10 +14,7 @@ my_array                            # array folder
          |_ ...
 ```
 
-`<timestamped_name>` has format `__timestamp_timestamp_uuid`, where:
-* `timestamp` is timestamp in milliseconds elapsed since 1970-01-01 00:00:00 +0000 (UTC)
-* `uuid` is a unique identifier
-
+Note: the `schema` [`<timestamped_name>`](./timestamped_name.md) does _not_ include the format version.
```

Comment:
I would add this note after "Any number of [array schema files](#array-schema-file)"...

And have something a little different... Maybe:
The schema file has name [`<timestamped_name>`](./timestamped_name.md)`. Note: the name does _not_ include the format version.